### PR TITLE
[Fixes #81] Add new component for social links

### LINKED ISSUE #81: Extract out social links into seperate component

### PR DESCRIPTION
**Pull request by sp35**
Fixes #81 
This creates a new component to separate the social links from the login component.

- [x] PR targets develop branch

### CODE REVIEW

File: src/app/@theme/components/social-links-component/social-links-component.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'intelowl-social-links-component',
+  templateUrl: './social-links-component.component.html',
+  styleUrls: ['./social-links-component.component.scss'],
+})
+export class SocialLinksComponentComponent implements OnInit {
```
Comment (Eshaan7) [inline src/app/@theme/components/social-links-component/social-links-component.component.ts:8]: no need for the `onInit` interface.

File: src/app/@theme/components/social-links-component/social-links-component.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      url: 'https://twitter.com/intel_owl',
+      target: '_blank',
+      icon: 'twitter',
+    },
+  ];
+
+  constructor() {}
+
+  ngOnInit(): void {}
```
Comment (Eshaan7) [inline src/app/@theme/components/social-links-component/social-links-component.component.ts:29]: remove this after removing the `onInit` interface.

File: src/app/@theme/components/social-links-component/social-links-component.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'intelowl-social-links-component',
```
Comment (Eshaan7) [inline src/app/@theme/components/social-links-component/social-links-component.component.ts:4]: can you just remove the `-component` from the name everywhere ? didn't notice it before but is pretty redundant.

File: src/app/@theme/components/social-links-component/social-links-component.component.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  templateUrl: './social-links-component.component.html',
+  styleUrls: ['./social-links-component.component.scss'],
+})
+export class SocialLinksComponentComponent {
```
Comment (Eshaan7) [inline src/app/@theme/components/social-links-component/social-links-component.component.ts:8]: should be `SocialLinksComponent`.

File: src/app/@theme/components/social-links-component/social-links-component.component.ts (hunk 3 above)
Comment (sp35) [reply src/app/@theme/components/social-links-component/social-links-component.component.ts:4]: My bad, it does look weird

**Comment by Eshaan7** [comment]
Thankyou!

**Pull request merged**